src/reporters/lang/en.js

```diff
@@ -181,6 +181,8 @@ const messages = {
   foundWarnings: 'Found $0 warnings.',
   foundErrors: 'Found $0 errors.',
 
+  noPackageJsonFound: 'No package.json found.',
```

Comment:
How about
```
notSavedLockfileNoDependencies: 'Lockfile not saved, no dependencies.',
```

We are not focusing on a package.json existence but on the packages that were installed